src/sionna/phy/fec/ldpc/encoding.py

```diff
@@ -39,23 +40,43 @@ class LDPC5GEncoder(Block):
         Precision used for internal calculations and outputs.
         If set to `None`, :py:attr:`~sionna.phy.config.precision` is used.
 
+    params_only: bool, (default `False`)
+        If set to `True`, the encoder will only initialize the parameters
+        needed for the encoding, but will not load the basegraph or perform
+        any further initialization. This is useful to obtain the parameters
+        of the encoder without initializing the full block.
+
+    allow_low_rates: bool, (default `False`)
```

Comment:
The reason we we don't allow low rates is that we don't support the 3GPP compliant rate-matching for such low rates, yet. 

My understanding of this flag is that it only disables the check and may produce a non-3GPP compliant code. This seems misleading to me and I would like to remove it.